src/libs/utils/validate.go

```diff
@@ -48,3 +51,20 @@ func ValidatePhoneOrEmail(loginValue string) bool {
 
 	return false
 }
+
+// IsStrongPassword checks if the password is strong enough
+func IsStrongPassword(password string) bool {
+	pwdbro := pwdbro.NewDefaultPwdBro()
```

Comment:
do not use this kind of packages, use a regex to validate the password, so that we can easily change the validator meeting project requirements.